docs/dev/stream/state/checkpointing.zh.md

```diff
@@ -76,6 +76,8 @@ Other parameters for checkpointing include:
 
   - *fail/continue task on checkpoint errors*: This determines if a task will be failed if an error occurs in the execution of the task's checkpoint procedure. This is the default behaviour. Alternatively, when this is disabled, the task will simply decline the checkpoint to the checkpoint coordinator and continue running.
 
+  - *is prefer checkpoint for recovery*: This determines if a job will fallback to checkpoint when there is a savepoint later than checkpoint.
```

Comment:
I dont know what's the difference between the checkpointing.zh.md and checkpointing.md files, do we need to change both?

Reply:
The files with pattern `xxx.zh.md` is copied for translating purpose. They will be translated into Chinese. So for the correctness, I think change both would be a good solution.